src/gsy_e/models/area/scm_manager.py

```diff
@@ -310,6 +313,22 @@ def gsy_energy_bill_excl_revenue_without_fees(self):
         return (self.gsy_energy_bill_excl_revenue - self.grid_fees - self.tax_surcharges
                 - self.fixed_fee - self.marketplace_fee - self.assistance_fee)
 
+    @property
+    def gsy_energy_bill_revenue(self):
+        """Energy bill of the home excluding revenue."""
```

Comment:
The docstring does not comply to the implementation. 
How about:
```suggestion
        """Total revenue from gird and community."""
```

Reply:
Good catch, thanks !